tests/clpy_tests/core_tests/test_carray.py

```diff
@@ -1,32 +1,42 @@
 import unittest
 
 import clpy
+from clpy.backend.opencl.exceptions import OpenCLProgramBuildError
 from clpy import testing
 
+import six
 
-class TestCArray(unittest.TestCase):
-
-    def test_size(self):
-        x = clpy.arange(3).astype('i')
-        y = clpy.ElementwiseKernel(
-            'raw int32 x', 'int32 y', 'y = x.size()', 'test_carray_size',
-        )(x, size=1)
-        self.assertEqual(int(y[0]), 3)
 
-    def test_shape(self):
-        x = clpy.arange(6).reshape((2, 3)).astype('i')
-        y = clpy.ElementwiseKernel(
-            'raw int32 x', 'int32 y', 'y = x.shape()[i]', 'test_carray_shape',
-        )(x, size=2)
-        testing.assert_array_equal(y, (2, 3))
+class TestCArray(unittest.TestCase):
 
-    def test_strides(self):
-        x = clpy.arange(6).reshape((2, 3)).astype('i')
-        y = clpy.ElementwiseKernel(
-            'raw int32 x', 'int32 y', 'y = x.strides()[i]',
-            'test_carray_strides',
-        )(x, size=2)
-        testing.assert_array_equal(y, (12, 4))
+    def test_size(self):  # TODO(vorj): support CArray::size()
```

Comment:
I confirm these tests fail. This problem should be fixed at #51 so it's fine in this PR.